includes/definitions/cyberpower.yaml

```diff
@@ -0,0 +1,13 @@
+os: cyberpower
+text: 'Cyberpower'
+type: power
+icon: cyberpower
+over:
+    - { graph: device_current, text: Current }
+    - { graph: device_voltage, text: Voltage }
+discovery:
+    - sysObjectId: .1.3.6.1.4.1.3808.1.1.3
+poller_modules:
+    ports: 0
```

Comment:
Out of curiosity, which firmware version was used here? It looks like [Cyberpower is updating the firmware regularly](https://dl4jz3rbrsfum.cloudfront.net/documents/CyberPower_RN_RMCARD205-FW-109.pdf), wonder if they know about the lack of SNMP port counters bug?